addons/EasyNuke/EasyNuke.lua

```diff
@@ -0,0 +1,207 @@
+--[[
+Copyright © 2018, FaceDesk Linkshell
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright
+notice, this list of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright
+notice, this list of conditions and the following disclaimer in the
+documentation and/or other materials provided with the distribution.
+* Neither the name of findAll nor the
```

Comment:
Change `findAll` here to `EasyNuke`.

Reply:
Done.  Guess it's easy to figure out where I copy/pasted it from. XD

Reply:
IANAL, but should be ok.